public/ipc.js

```diff
@@ -157,18 +157,34 @@ ipc.on('get job', async (event, data) => {
 })
 
 ipc.on('abort job', (event, data) => {
-  Object.values(queues[data.job.queue.name].childPool.retained).forEach((child) => {
-    child.send({__abortJobId__: data.job.id})
+  dialog.showMessageBox(
+    {type: 'question',
```

Comment:
Let's style the object as follows for consistency:

```
{
  key: val,
  key, val,
  ...
}
```

instead of:

```
{key: val,
 key: val,
  ...
}
```